Makefile

```diff
@@ -72,25 +70,18 @@ docker.io/%:
 	$(MAKE) BUILD_IMAGE="$@" build
 
 .PHONY: checkout
-checkout: checkout-containerd checkout-runc
-
-.PHONY: checkout-containerd
-checkout-containerd: src
+checkout: src
 	./scripts/checkout.sh src/github.com/containerd/containerd "$(REF)"
-
-# this must be a separate target, otherwise "RUNC_REF" is not evaluated correctly
-.PHONY: checkout-runc
-checkout-runc: checkout-containerd
-	./scripts/checkout.sh src/github.com/opencontainers/runc "$(RUNC_REF)"
+	./scripts/checkout.sh src/github.com/opencontainers/runc "$(shell ./scripts/determine-runc-version)"
```

Comment:
I was thinking something like this :
```
./scripts/checkout.sh src/github.com/opencontainers/runc `./scripts/determine-runc-version`
```

Reply:
Yeah, trying to avoid the backticks as they aren't posix (and IIRC officially deprecated? (at least "discouraged")), but using a plain `$()` doesn't work in Make without using `$(shell ...)` 😞

Reply:
You might wanna try `$$(shell command)`

Reply:
And it turns out it correctly uses RUNC_REF from `make RUNC_REF=1.2.3.4`. Btw, I'd suggest echoing the selected RUNC_REF in `determine-runc-version`.

Reply:
> You might wanna try `$$(./scripts/determine-runc-version)`

🤦 you're right. for some reason I was thinking that was how to handle it in Jenkinsfile, not in Makefile


> Btw, I'd suggest echoing the selected RUNC_REF in `determine-runc-version`.

Not sure I understand what you mean; it should already print the version it found, and print an informational message on `stderr` but perhaps that won't be visible due to the sub shell; https://github.com/docker/containerd-packaging/blob/6a2e254c9b238a482c5bdebe8e5d99386522515a/scripts/determine-runc-version#L26-L43

Reply:
Oh! You mean print the version as part of the `INFO:` messages? Yes, that makes sense